code/game/objects/items.dm

```diff
@@ -446,17 +462,18 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 			inspec += "\n<b>MIN.STR:</b> [minstr]"
 		
 		if(force)
-			inspec += "\n<b>FORCE:</b> [get_force_string(force)]"
+			inspec += "\n<b>FORCE:</b> [get_force_string(force)] ([force])"
 		if(gripped_intents && !wielded)
 			if(force_wielded)
-				inspec += "\n<b>WIELDED FORCE:</b> [get_force_string(force_wielded)]"
+				inspec += "\n<b>WIELDED FORCE:</b> [get_force_string(force_wielded)] ([force_wielded])"
```

Comment:
this is not

due to sharpness + str + damfactors the code'll just be deceiving players cus numbers will not match up with damage received even if they self-hit to test

even switching to strong intent will vary the numbers